### DISCUSSION TITLE
Dependencies pandas and numpy don't install with pip on Termux

### DISCUSSION THREAD

**Discussion by Nightmare4013** [sherlock-project/sherlock#2055] [Q&A] [answered]
Every time I try to download requirements it says "this error originates from subprocesses and is not a problem with pip" how can I fix it

**Comment by matheusfelipeog** [comment]
Hi @Nightmare4013. Are you using Termux?

**Reply by Nightmare4013** [reply]
Yes I'm using termux and I've already seen your answer to someone who previously mentioned this issue it was helpful.

**Reply by matheusfelipeog** [reply, marked as answer]
Alright, just to facilitate potential searches, I'll also link it here: https://github.com/sherlock-project/sherlock/issues/1945#issuecomment-1850757406

If the issue has been resolved, please mark the answer to be highlighted for other users.
